tests/topotests/bgp_bmp/r1/zebra.conf

```diff
@@ -0,0 +1,10 @@
+hostname r1
```

Comment:
Same here, can you drop hostname, and password configurations?

Reply:
done!